### PR TITLE
C2LC-360: Remove fixed width for coordinate input text box and limit …

### PR DESCRIPTION
**Pull request by chosww**
…character input length to 2

### CODE REVIEW

**Comment by chosww** [comment]
Hey @the-t-in-rtf, could you do some device testings for this PR please?

**Comment by the-t-in-rtf** [comment]
This looks good on the simulated devices I've tried.  I tested the smallest iPad Pro (9.7") and iPad Air.  I saw the problem with [the live site](https://code.weavly.org/) on both devices at all levels of text zoom.  With your fix, I don't see the problem regardless of how far I zoom in and out.  So this looks good to me.

File: src/ProgramBlockEditor.js [hunk 1]
```diff
@@ -767,6 +767,8 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
                             className={characterPositionTextInputClassName}
                             aria-label={this.props.intl.formatMessage({id:'ProgramBlockEditor.editPosition.columnPosition'})}
                             aria-disabled={this.props.editingDisabled}
+                            maxLength='2'
```
Comment (the-t-in-rtf) [inline src/ProgramBlockEditor.js:770]: Is this just to make them match, or are you future-proofing?  We don't have any double character column positions.

Comment (the-t-in-rtf) [reply src/ProgramBlockEditor.js:770]: I can see that having a `maxLength` at all is an improvement over what's deployed to production.  Can we try this with a `maxLength` of `1`?

Comment (chosww) [reply src/ProgramBlockEditor.js]: Good idea, I changed maxLength of column text box to be 1, and left size to be 2 for its width.

Comment (the-t-in-rtf) [reply src/ProgramBlockEditor.js]: With these changes I think this is ready for further review.

**Pull request merged**